homework.md

```diff
@@ -4,110 +4,110 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+Red
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS used to define style for web pages including layout, design and variations
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language commonly used in web development
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Help to share information
+Enable people to make their own changes
+Allow us to make checkpoints so that we can save our work as we go along
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
```

Comment:
So I love how you were descriptive in your cloning, because on this one where are we copying it? The key idea here is directly being as descriptive as possibly like you had done in the cloning section